packages/suite/src/reducers/suite/metadataReducer.ts

```diff
@@ -57,6 +58,7 @@ const metadataReducer = (state = initialState, action: Action): MetadataState =>
                 draft.providers.push(action.payload);
                 break;
             case METADATA.REMOVE_PROVIDER:
+                // todo: identification should be dataType + clientId
```

Comment:
Is this TODO something that needs to be done before we merge this?

Reply:
more likely unfinished. at the moment, it is not needed because each feature (passwords, labels) have distinct provider. In case we wanted to support 2 different features in 1 provider. we would need to add this?  I think this comment can be removed now 🤔

Reply:
Or maybe this what you have just explained here shall be put into that comment.

Reply:
ad858d05970d9d953820a922133ccb052076ac14